tests/e2e.test.js

```diff
@@ -260,7 +261,7 @@ describe('Search', () => {
 describe('Ordering', () => {
   const icons = Object.values(simpleIcons);
   const titles = icons.map((icon) => icon.title);
-  const hexes = icons.map((icon) => icon.hex).sort(sortColors);
+  const hexes = sortByColors(icons.map((icon) => icon.hex));
```

Comment:
It's fine either way but any particular reason you didn't implement `sortByColors` so that it can be used as `arr.sort(sortByColors)`?

Reply:
I need full control of the array to distribute them in buckets.
It may be possible to implement it as a stateless sorting function, but that was not the way it started. If it is absolutely necessary I can try it, but it will be significantly harder than what is already done.

Reply:
Makes sense, in that case just leave it as is.